### DISCUSSION TITLE
Attribute option for mushroom entity card

### DISCUSSION THREAD

**Discussion by MagieMalone** [piitaya/lovelace-mushroom#293] [Ideas]
As the title suggest, make it possible to display an entity's attribute instead of the state, like you can do with the default entity card

**Comment by piitaya** [comment]
The proper way to do this is to use template card : https://github.com/piitaya/lovelace-mushroom/issues/253
We do not propose attributes because attributes are raw data (not translated) and different for all entities. Template are the good way to format the attribute the way you want.
